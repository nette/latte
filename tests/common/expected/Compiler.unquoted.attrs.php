<?php
%A%
final class Template%a% extends Latte\Runtime\Template
{

	public function main(array $ÊŸ_args): void
	{
%A%
		echo '<span ';
		echo LR\AttributeHandler::formatHtmlAttribute('title', $x) /* line %d%:%d% */;
		echo ' ';
		echo LR\AttributeHandler::formatHtmlAttribute('class', $x) /* line %d%:%d% */;
		echo '></span>

<span ';
		echo LR\AttributeHandler::formatHtmlAttribute('title', $x) /* line %d%:%d% */;
		echo ' ';
		echo LR\Filters::escapeHtmlTag($x) /* line %d%:%d% */;
		echo '></span>

<span title="';
		if (true) /* line %d%:%d% */ {
			echo LR\Filters::escapeHtmlAttr($x) /* line %d%:%d% */;
		} else /* line %d%:%d% */ {
			echo 'item';
		}
		echo '"></span>

<span ';
		echo LR\Filters::escapeHtmlTag('title') /* line %d%:%d% */;
		echo '="';
		echo LR\Filters::escapeHtmlAttr($x) /* line %d%:%d% */;
		echo '"></span>

<span attr="c';
		echo LR\Filters::escapeHtmlAttr($x) /* line %d%:%d% */;
		echo 'd"></span>

<span ';
		echo LR\AttributeHandler::formatHtmlAttribute('onclick', $x) /* line %d%:%d% */;
		echo ' ';
		echo LR\Filters::escapeHtmlTag($x) /* line %d%:%d% */;
		echo '></span>

<span onclick="c';
		echo LR\Filters::escapeHtmlAttr(LR\Filters::escapeJs($x)) /* line %d%:%d% */;
		echo 'd"></span>

<span attr';
		echo LR\Filters::escapeHtmlTag($x) /* line %d%:%d% */;
		echo 'b="c';
		echo LR\Filters::escapeHtmlAttr($x) /* line %d%:%d% */;
		echo 'd"></span>
';
	}
}
