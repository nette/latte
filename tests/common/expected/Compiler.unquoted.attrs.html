<span title='&apos; &amp; "' class='&apos; &amp; "'></span>

<span title='&apos; &amp; "' &apos;&#32;&amp;&#32;&quot;></span>

<span title="&apos; &amp; &quot;"></span>

<span title="&apos; &amp; &quot;"></span>

<span attr="c&apos; &amp; &quot;d"></span>

<span onclick='&apos; &amp; "' &apos;&#32;&amp;&#32;&quot;></span>

<span onclick="c&quot;&apos; &amp; \&quot;&quot;d"></span>

<span attr&apos;&#32;&amp;&#32;&quot;b="c&apos; &amp; &quot;d"></span>